packages/web/package.json

```diff
@@ -1,48 +1,54 @@
 {
-  "name": "@foton/web",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@types/jest": "23.3.12",
-    "@types/node": "10.12.18",
-    "@types/react": "16.7.18",
-    "@types/react-dom": "16.0.11",
-    "add": "^2.0.6",
-    "apollo-boost": "^0.1.23",
-    "apollo-client": "^2.4.8",
-    "apollo-client-preset": "^1.0.8",
-    "apollo-link-context": "^1.0.12",
-    "apollo-link-http": "^1.5.9",
-    "apollo-link-ws": "^1.0.12",
-    "apollo-utilities": "^1.0.27",
-    "graphql-tag": "^2.10.0",
-    "react": "^16.7.0",
-    "react-apollo": "^2.3.3",
-    "react-dom": "^16.7.0",
-    "react-router-dom": "^4.3.1",
-    "react-scripts": "2.1.3",
-    "styled-components": "^4.1.3",
-    "typescript": "3.2.2",
-    "yarn": "^1.13.0",
-    "jest": "23.6.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "now-build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ],
-  "devDependencies": {
-    "@types/react-router-dom": "^4.3.1"
-  }
-}
\ No newline at end of file
+    "name": "@foton/web",
+    "version": "0.1.0",
+    "private": true,
+    "dependencies": {
+        "@types/jest": "23.3.12",
+        "@types/node": "10.12.18",
+        "@types/react": "16.7.18",
+        "@types/react-dom": "16.0.11",
+        "@types/yup": "^0.26.9",
```

Comment:
types are devDeps

Reply:
_create-react-app_ came with some default types as _dependencies_ if we move all the types to _devDependencies_ wouldn't break the app?

Reply:
Actually, types are needed for development and aren't used in runtime, so now all the types are as _devDependencies_! Fixed!